to-do-list/node_modules/todomvc-app-css/index.css

```diff
@@ -0,0 +1,393 @@
+@charset 'utf-8';
```

Comment:
katalog `node_modules` nie powinien być wrzucony na repo. package-lock.json mówi o tym jakie dokładnie wersje bibliotek są potrzebne, także każdy u siebie lokalnie może je sobie zainstalować.

Dzięki temu nie dublujemy tego kodu, który i tak jest na repozytoriach bibliotek. Jest dzięki temu czytelniej, bo wiadomo, że na repo jest tylko kod, za który my odpowiadamy i którego mamy własność. Plus w node_modules często nie ma plików źródłowych bibliotek, tylko zbundle'owane moduły. Na przykład jeśli biblioteka jest pisana w TS, to może się okazać, ze to, co ląduje w node_modules jak zainstalujesz ją przez npm to tylko zbudowane JSowe pliki.

Poza tym node_modules potrafią się bardzo rozroznąć 😄 (nierzadko >1GB w większych projektach)

![image](https://github.com/user-attachments/assets/02ba8003-a032-4e7a-a166-5a6aabbad064)

Reply:
także trzeba ten katalog po prostu dodać do .gitignore